swap/programs/swaps/src/verifying_key_swaps.rs

```diff
@@ -31,13 +31,13 @@ pub const VERIFYINGKEY_SWAPS: Groth16Verifyingkey = Groth16Verifyingkey {
     ],
 
     vk_delta_g2: [
-        22, 55, 117, 213, 81, 56, 115, 140, 165, 134, 27, 175, 25, 186, 168, 156, 213, 118, 53, 73,
-        20, 203, 43, 137, 51, 27, 252, 148, 24, 255, 31, 198, 23, 87, 190, 56, 191, 223, 149, 63,
-        87, 123, 110, 123, 153, 207, 159, 204, 78, 70, 190, 78, 197, 40, 217, 115, 12, 150, 59,
-        136, 167, 226, 95, 14, 6, 148, 5, 12, 180, 145, 0, 95, 153, 33, 36, 246, 93, 31, 34, 35,
-        51, 163, 5, 57, 190, 224, 189, 34, 186, 150, 5, 154, 41, 174, 68, 48, 42, 73, 167, 16, 212,
-        72, 19, 120, 97, 13, 76, 235, 124, 116, 201, 60, 16, 44, 156, 12, 178, 191, 27, 125, 125,
-        183, 186, 143, 19, 207, 134, 32,
```

Comment:
same here we push a verifying key but not the proving key

Reply:
[Removed it + added to .gitignore](https://github.com/Lightprotocol/psp-examples/pull/8/commits/6a88a0fe963013e406c624230d45f8bccab735f7)